script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -630,7 +631,7 @@ There had never been a case of someone inverting during childhood, at least not
 [sha:767694a9d589b11f4aec9f09ae2df798b939a907]{
 -- Page 42, Offset 31789.
 -- 「遠野の人間が持つ異能にはそれぞれ個人差があるそうです。外見がまったく変わらないモノがいれば、本当に体の形が変わってしまうモノもいる。
-"The abilities possessed by the members of the Tohno family vary widely from person to person.
+"It seems that there are individual differences in each and every Tohno Human's different abilities.
```

Comment:
Original reads significantly better to me

Reply:
I think the original reads better but this version seems to be closer to what she's saying. lit. "the special powers that human tohnos possess each have individual differences". 

Moreover,  それぞれ個人差があるそうです reads to me like they're not _great_ differences, almost closer to minutia?

That said, I'm not going to fight against reverting because I don't really think the distinction matters very much and the old version is nicer

Reply:
I would revert and remove widely